scripts/build_docs.sh

```diff
@@ -0,0 +1,45 @@
+# Build HTML documentation for FLT
+# The output will be located in docs/docs
+
+# Template lakefile.toml
+template() {
+  cat <<EOF
+name = "docbuild"
+reservoir = false
+version = "0.1.0"
+packagesDir = "../.lake/packages"
+
+[[require]]
+name = "FLT"
+path = "../"
+
+[[require]]
+scope = "leanprover"
+name = "doc-gen4"
+rev = "TOOLCHAIN"
+EOF
+}
+
+# Create a temporary docbuild folder
+mkdir -p docbuild
+
+# Equip docbuild with the template lakefile.toml
+template > docbuild/lakefile.toml
+
+# Substitute the toolchain from lean-toolchain into docbuild/lakefile.toml
+sed -i s/TOOLCHAIN/`grep -oP 'v4\..*' lean-toolchain`/ docbuild/lakefile.toml
+
+# Initialise docbuild as a Lean project
+cd docbuild
+~/.elan/bin/lake update FLT
```

Comment:
Pietro, if you want to keep `MATHLIB_NO_CACHE_ON_UPDATE`, it's here:
```suggestion
MATHLIB_NO_CACHE_ON_UPDATE=1 ~/.elan/bin/lake update FLT
```
But do you want it? What does it even do?

Reply:
Unfortunately it's necessary to avoid `lake exe cache get` to run after the update and make the docbuild setup work.

Se [here](https://leanprover.zulipchat.com/#narrow/stream/270676-lean4/topic/New.20recommended.20default.20setup.20for.20doc-gen4) for further details.

Reply:
But wait. Did the workflow run without any issues?

Reply:
Ah right, it just gets rid of a non-blocking error message. I guess we can keep it

Reply:
Yes, agreed.